packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -156,6 +156,8 @@ async function startCompass(
   addCommands(app);
   addDebugger(app);
 
+  app.wrappedClient = wrapCommands(app);
```

Comment:
Would be nice to avoid having this special `wrappedClient`

Reply:
Any ideas?

Reply:
Similar to `addDebugger` let's patch the prototype directly. I think in this case where we are not altering method behavior in any way that would be a problem for people who will write tests it is totally fine to do